content/en/docs/collector/internal-telemetry.md

```diff
@@ -144,6 +144,62 @@ journalctl | grep otelcol | grep Error
 
 {{% /tab %}} {{< /tabpane >}}
 
+### Configure internal traces
+
+The Collector does not expose traces by default, but can be configured to.
+
+{{% alert title="Caution" color="warning" %}}
+
+Internal tracing is an experimental feature, and no guarantees are made as to
+the stability of the emitted span names and attributes.
+
+{{% /alert %}}
+
+The following configuration can be used to emit internal traces from the
```

Comment:
We also support the `level` option for traces. @codeboten any wording suggestions on what to put there?

Reply:
I think we should leave the level option out because it is likely to be removed in the future

Reply:
Makes sense!